src/translate.c

```diff
@@ -83,13 +83,6 @@ void readInp(t_trans *tdat,t_ham *ham,t_files *FH){
     fread(&ham->t,sizeof(int),1,FH->IE);
     N=tdat->singles*(tdat->singles+1)/2;
     fread(&ham->He[0],sizeof(float),N,FH->IE);
-    //    for (k=0;k<tdat->singles;k++){
-      //      for (l=k;l<tdat->singles;l++){
```

Comment:
Why did you take this code out?

Reply:
I had to change something to illustrate a change from coffee code club. We should clean up this translate code, but it probably belongs in a different issues.